README.md

```diff
@@ -137,11 +135,17 @@ You can deploy this project using:
 - Twitter: [@twitterhandle](https://twitter.com/mwovi_peter)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/peter-mwovi-57141a179/)
 
-👤 **Juan Carlos Muñoz**
+👤 **Arslan Majeed**
+
+- GitHub: [@githubhandle](https://github.com/arslan49sp)
+- Twitter: [@twitterhandle](https://twitter.com/arslan49sp)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/arslan49sp)
```

Comment:
Kindly fix the broken link in the `Authors` section, Let's avoid broken links in our readme file for a professional repository. 🛠🤓